### PR TITLE
test domain save from interactive training - merge

### PR DESCRIPTION
**Pull request by tzolkincz**
**Proposed changes**:
- PR for merge as discussed in #1353 (I've deleted source branch and now cannot reopen the original PR)

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by Ghostvv** [comment]
Great! thanks, now let's see if test is passing

**Comment by Ghostvv** [comment]
For some reason other than py 3.6 tests do not pass (we're looking into it), but your test seems to be passing now. Could you please add assert that other default actions are not present as well

**Comment by tzolkincz** [comment]
Sure, I'll add other asserts as well.

File: tests/test_interactive.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from rasa_core import utils
 from rasa_core.training import interactive
 from rasa_core.utils import EndpointConfig
+from rasa_core.actions import action
```
Comment (Ghostvv) [inline tests/test_interactive.py]: let's import the function
```suggestion
from rasa_core.actions.action import default_actions
```

File: tests/test_interactive.py [hunk 2]
```diff
@@ -205,3 +206,25 @@ def test_undo_latest_msg(mock_endpoint):
...
+
+    events = tracker_json.get("events", [])
+
+    domain_path = tmpdir.join("interactive_domain_save.yml")
+    interactive._write_domain_to_file(domain_path, events, mock_endpoint)
+
+    saved_domain = utils.read_yaml_file(domain_path)
+
+    httpretty.disable()
+    for default_action in action.default_actions():
```
Comment (Ghostvv) [inline tests/test_interactive.py]:
```suggestion
    for default_action in default_actions():
```

**Pull request merged**